sdk/src/main/java/com/akkaserverless/javasdk/eventsourcedentity/EventSourcedEntityBase.java

```diff
@@ -79,6 +96,8 @@ protected EventContext eventContext() {
 
       OnSuccessBuilder<S> emitEvent(Object event);
 
+      OnSuccessBuilder<S> emitEvents(Collection<? extends Object> event);
```

Comment:
Emitting multiple events needed to live up to TCK test.

Reply:
Could have some more variants and avoid `Colleciton`? 

```java
emitEvent(Object event);
emitEvent(Object event1, Object event2);
emitEvent(Object event1, Object event2, Object event3);
emitEvent(Object event1, Object event2, Object event3, Object* otherEvents);
```

Reply:
I think I agree, but I don't want to change it here, so let's revisit that later.

Reply:
That was wrt persist vs emit. The vararg overload I added now.